src/ui/Windows/PasskeyEntry.cpp

```diff
@@ -259,13 +274,6 @@ BOOL CPasskeyEntry::OnInitDialog(void)
     return FALSE;
   }
 
-  // Following works fine for other (non-hotkey) cases:
-  if (m_index == GCP_RESTORE || m_index == GCP_WITHEXIT) {
-    SetWindowPos(&CWnd::wndTopMost, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE);
-    SetActiveWindow();
-    SetForegroundWindow();
-  }
```

Comment:
This blocks seems to have been removed completely - why?

Reply:
It was redundant.